### ISSUE TITLE
Screen reader is not announcing the search results information after searching in country search field.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#11854]
### Link to reproducible scenario

https://www.ag-grid.com/javascript-data-grid/filtering/

### Describe the bug

**Test Environment:**
OS: Windows 11[Version 24H2(OS Build 24631.3593)]
Edge: Version 135.0.2535.51 (Official build) (64-bit)
NVDA:2024.1(2024.1.0.31547)
Test URL: https://www.ag-grid.com/javascript-data-grid/filtering/

**User Impact:**
Screen reader users may find it difficult to understand the purpose and functionality if screen reader doesn't announce search result information. 

**Pre-Requisites:**
1. Turn on NVDA/Jaws
2. For Browse mode - use up/down arrow keys.
3. Use Insert + Space bar for changing the modes i.e., Browse to Focus or Focus to Browse.

**Repro Steps:**

1. Open the application using URL: https://www.ag-grid.com/javascript-data-grid/filtering/ in windows.
2. Column filter page will get open.
3. Press tab key to move to the "Example: Server-Side Pagination"
4. Press tab key to move to the Country control and activate it using enter key.
5. Observe that, Screen reader is announcing the search results information after searching in country search field or not.

**Actual Behavior:**
After entering the data in country search field, screen reader is not announcing the search results information.
Screen reader remains silent

**Refer Attachment:** 
1.Screen reader is not announcing the search results information after searching in country search field..png
2.Screen reader is not announcing the search results information after searching in country search field..mp4

<img width="978" height="561" alt="Image" src="https://github.com/user-attachments/assets/5280f6c4-3b1a-4c87-a9bf-a83808612f98" />
https://github.com/user-attachments/assets/de82e055-9f47-4183-80e5-b8d5076ba161

**Expected Behavior:**
After entering the data in country search field, screen reader should announce the search results information.

### Version

34.0.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team,
As this issue is still repro'ing. please provide an update on this issue.
